CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 ## 0.8.0 (YYYY-MM-DD)
 
 ### Breaking Changes
-* None.
+* Reworked configuration hierarchy:
+  * Separated common parts of `RealmConfiguraion` and `SyncConfiguration` into `io.realm.Configuration` to avoid poluting the base configuration with local-only options.
```

Comment:
```suggestion
  * Separated common parts of `RealmConfiguraion` and `SyncConfiguration` into `io.realm.Configuration` to avoid polluting the base configuration with local-only options.
```